web/server/vue-cli/src/components/Product/ProductNameColumn.vue

```diff
@@ -124,7 +131,22 @@ export default {
       ]
     };
   },
+  computed: {
+    confidentialityIcon() {
+      return this.confidentialityIconDefs[
+        this.encodeValue(this.product.confidentiality)
+      ];
+    }
+  },
   methods: {
+    encodeValue(confidentialityId) {
+      return this.confidentialityFromCodeToString(confidentialityId);
+    },
+
+    decodeValue(confidentialityName) {
+      return this.confidentialityFromStringToCode(confidentialityName);
+    },
```

Comment:
Why do we need these functions?  For example instead of using `encodeValue` function we can use the `confidentialityFromCodeToString` function.